src/components/Form/Form.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { UserType } from '../Todo';
+
+class Form extends React.Component {
+  state = {
+    title: '',
+    userId: '',
+    titleValid: true,
+    userIdValid: true,
+  }
+
+  handleChange = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({ [name]: value });
+
+    switch (name) {
+      case 'title':
+        this.setState({
+          titleValid: true,
+        });
+        break;
+      case 'userId':
+        this.setState({
+          userIdValid: true,
+        });
+        break;
+      default:
+        break;
+    }
+  }
+
+  checkTodo = (userId, title) => {
+    this.setState({
+      titleValid: /^\w+$/.test(title),
+      userIdValid: userId,
+    });
+
+    if (this.state.title !== '' && this.state.userId !== ''
```

Comment:
```suggestion
    if (title.length > 0 && userId
```
деструктуризируй значения из state